### ISSUE TITLE
support ActiveX?

### ISSUE THREAD

**Issue by mqliang** [ruffle-rs/ruffle#6298]
I have a very legacy player (let me call it "legacy-player"), which can decrypt a SWF file then call ActivateX to play. Please note that the SWF file is encrypted so it can not be played by a regular standard flash player directly. It must be opened by "legacy-player", and let the "legacy-player" to call ActivateX.

So Just wanna know is there any plan to support ActivateX?

**Comment by mqliang** [comment]
> When you say "ActivateX" are you talking about [ActiveX](https://en.wikipedia.org/wiki/ActiveX) or something else?

Yeah, I mean [ActiveX](https://en.wikipedia.org/wiki/ActiveX), sorry for the typo.

**Comment by mqliang** [comment]
> To decrypt the SWF, I suggest running it in your "legacy player", then using the "Search SWFs in memory" feature of [JPEXS Decompiler](https://github.com/jindrapetrik/jpexs-decompiler/).

Tried the "Search SWFs in memory" feature of [JPEXS Decompiler](https://github.com/jindrapetrik/jpexs-decompiler/). It turns out that a single SWF file will be decrypted by the "legacy player" into multiple files (some of them are SWF files and some not). "Search SWFs in memory" can only find out the SWF files, it can not extract other parts.

In addition, I believe there are some controlling logic built in the "legacy player" to control the animation. 

To continue use the "legacy-player", I currently have a VM with flash-activex installed. If ruffle has a plan of supporting activex, that will be great.

**Comment by Poopooracoocoo** [comment]
Reopening as it looks to have been a mistake

**Comment by mero09-99** [comment]
This should be resolved
